notebooks/official/generative_ai/distillation.ipynb

```diff
@@ -0,0 +1,632 @@
+{
```

Comment:
<p>Suggest to change:</p><ul><li>'Distillation Step by Step' &gt; the official term 'distilling step by step"</li><li>Vertex AI platform &gt; Vertex AI</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2656/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
done.